python-package/lightgbm/libpath.py

```diff
@@ -20,7 +20,8 @@ def find_lib_path():
         dll_path.append(os.path.join(curr_path, '../../windows/x64/Dll/'))
         dll_path.append(os.path.join(curr_path, './windows/x64/Dll/'))
         dll_path.append(os.path.join(curr_path, '../../Release/'))
-        dll_path = [os.path.join(p, 'lib_lightgbm.dll') for p in dll_path]
+        dll_path.append(os.path.join(curr_path, '../../windows/x64/Release/')) 
+        dll_path = [os.path.join(p, 'lightgbm.dll') for p in dll_path]
```

Comment:
That my mistake，I am so sorry.
But I think that the installation guide says: " Set configuration to Release and x64 (set to DLL for building library) . "
might not very clear for beginner, because people who want to use an python version only want the DLL, so this way might be better:" Set configuration to DLL and x64 to build library (or set to Release for building the .exe) . "
Again, sorry for my mistake.